person.rb

```diff
@@ -1,21 +1,27 @@
 require './nameable'
+require './rental'
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] the` add-rental` should take **date** and **book** as parameters and also should be passed to the new rental every time you add a new book.